### DISCUSSION TITLE
Unable to jump to artist on SB Radio when playing songs via spotty

### DISCUSSION THREAD

**Discussion by rosscullen** [LMS-Community/slimserver#1089] [General]
I have notice an issue on SB Radio when playing anything on Spotty. When I press and hold the scrollwheel and navigate down to "Artist: [Artistname]" and select it, it won't bring me to that artist. 

If I try the same using a song from my local library, it works fine. Any suggestions? I am using latest version of Lyrion and a recent custom firmware for SB Radio. Thanks

**Comment by michaelherger** [comment]
That feature only exists for items which are indexed into your music library: the menu item would refer to this artist in your library. 

But there should be "On Spotify..." links taking you to the artist on Spotify. Can you confirm?

**Comment by rosscullen** [comment]
@michaelherger thanks for reply. 

Yes, the artist(s) are in my local library. For example, if I go into Spotty and play artist "Alien Ant Farm". While playing, I hold down the scroll wheel and navigate to "Artist: Alien Ant Farm" (which is in my local library). I select it but it just shakes (eg. it does not move off this screen).

I can confirm that I can select the "on Spotify: Alien Ant Farm" option and that works. 

It would be great to get it working as sometimes I come across a song on Spotify and I like to check my local library to see what I have stored locally for this artist. Thanks

**Comment by michaelherger** [comment]
Can you further specify: the track you're playing, is it a local track, or a remote track imported to your library? Would you mind sharing your server.log.zip (Settings/Information) with me?

**Comment by michaelherger** [comment]
Ok, re-reading your last comment I understand it's a Spotify track. And yes, I can reproduce this. I'm surprised... Will look into this! Thanks for the heads up!

**Reply by rosscullen** [reply]
Phew! Thank goodness, i'm not going crazy 😥 Glad to hear you were able to reproduce @michaelherger

**Reply by rosscullen** [reply]
Hi @michaelherger gentle reminder on above (to keep it active)... appreciate you are really busy with the recent events and subsequent changes. Thanks 👍🏻

**Reply by michaelherger** [reply]
Oh, I actually addressed this in #1095: LMS would link to the artist _if_ it was known in the local library. In other cases you'd have to use the "On Spotify" menu to navigate to the artist on Spotify specifically.

**Reply by rosscullen** [reply]
Thanks @michaelherger had a quick look and works really well.  Many thanks as ever 😊👍
